### PR TITLE
Ia catalog

### PR DESCRIPTION
**Pull request by nikolapejcic**
Added some basic links and pages including "Button"

### CODE REVIEW

**Comment by theinterned** [comment]
@nikolapejcic do you think it would make more sense to switch the order of **Foundation** and **Getting Started** sections?

**Comment by theinterned** [comment]
discussed IRL with @nikolapejcic we agreed that given the early state of the project Getting Started would be at the top as we expect to be on boarding more users — as the project matures we will move it to the bottom.

**Comment by theinterned** [comment]
@nikolapejcic I added the rest of the pages from your initial IA. 

I rearanged them a bit. Have a look

**Review by theinterned — APPROVED** [review]
Okay, I've added the rest of the content and made a few adjustments. 

I also moved the "pages" config out into it's own `routes.js` file as it was getting quite large.

@nikolapejcic have a look and if you approve I'll go ahead and merge.

**Comment by theinterned** [comment]
@nikolapejcic and I resolved outstanding discussions IRL

File: docs/catalog/guidlines/typography.md [hunk 1]
```diff
@@ -1 +1,53 @@
-Coming soon.
+## Typeface
```
Comment (theinterned) [inline docs/catalog/guidlines/typography.md:1]: are these doc changes for type meant to be in this PR?

File: sandbox/src/Nikola/Bgrs.story.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import styles from "./Bgrs.scss";
```
Comment (theinterned) [inline sandbox/src/Nikola/Bgrs.story.js:4]: As `Bgrs.scss` doesn't actually export anything the correct why to load it is using a non-assigning import:

```js
import './Bgrs.scss';
```

File: sandbox/src/Nikola/Bgrs.scss [hunk 3]
```diff
@@ -0,0 +1,884 @@
+@import 'Bgrs/reset';
+@import 'Bgrs/variables';
+@import 'Bgrs/typography';
+@import 'Bgrs/utilities';
+
+.bgrs{
+  .nav-bar{
```
Comment (theinterned) [inline sandbox/src/Nikola/Bgrs.scss:7]: why the nested selectors?

File: sandbox/src/Nikola/Bgrs.scss [hunk 4]
```diff
@@ -0,0 +1,884 @@
...
+
+.button{
+  &__primary{
+    background: $blue-dark;
+    border: solid 1px $blue-dark;
+    color: $white;
+    padding: 4px;
+    border-radius: 2px;
+  }
+}
```
Comment (theinterned) [inline sandbox/src/Nikola/Bgrs.scss:922]: almost 1k lines of scss!

File: sandbox/src/Nikola/Bgrs/_reset.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+body{
```
Comment (theinterned) [inline sandbox/src/Nikola/Bgrs/_reset.scss:1]: @nikola-nulogy @matthewlyle what do you say? is this a good enough general reset? Maybe we should have a `@nulogy/reset.css` that we can use everywhere?

File: sandbox/src/Nikola/Bgrs/_utilities.scss [hunk 6]
```diff
@@ -0,0 +1,4 @@
+.unlist{
```
Comment (theinterned) [inline sandbox/src/Nikola/Bgrs/_utilities.scss:1]: if you are going to use the `@extend` keyword, to better control the weird hoisting issues, it's a good idea to make this an [`%placeholder {}` selector](https://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_foo) and make sure you import these files first. 

```scss
%unlist{
  // ...
}

.my-list {
  @extend %unlist;
  // ...
}

[source](https://sass-guidelin.es/#extend)

File: docs/catalog/guidlines/typography.md (hunk 1 above)
Comment (theinterned) [reply docs/catalog/guidlines/typography.md:1]: I looks like there are several commits from @matthewlyle about type scales in this PR. Is this on  purpose?

**Comment by nikola-nulogy** [comment]
This PR is not relevant anymore.

**Pull request merged**